src/lib/notifications.ts

```
@@ -0,0 +1,60 @@
+        throw new Error('Project ID not found');
+      }
+      token = (
+        await Notifications.getExpoPushTokenAsync({
+          projectId,
+        })
+      ).data;
+    } catch (e) {
+      token = `${e}`;
+    }
+  } else {
+    console.log('Must use physical device for Push Notifications');
```

ESLint

[error] Delete `;`

(prettier/prettier)